tests/integration/cattletest/core/test_whitelist_default_registries.py

```diff
@@ -174,7 +176,7 @@ def test_registries(super_client, client, context,
 
     uuid1 = "ubuntu:v1"
     lc_registry = {"imageUuid": uuid1}
-    launch_config_upgrade = {"imageUuid": "sim:registry/ubuntu:v1"}
+    launch_config_upgrade = {"imageUuid": "sim:registry.com/ubuntu:v1"}
     service = client.create_service(name=random_str(),
```

Comment:
`registry.com` should be regarded as registry too. If we use `registry`, it will be parsed with default `index.docker.io`.